flask_jwt_extended/view_decorators.py

```diff
@@ -118,7 +119,7 @@ def wrapper(fn):
         @wraps(fn)
         def decorator(*args, **kwargs):
             verify_jwt_in_request(optional, fresh, refresh, locations)
-            return fn(*args, **kwargs)
+            return current_app.ensure_sync(fn)(*args, **kwargs)
```

Comment:
Does this break 1.x version of flask?

Reply:
oh hell. Yeah, it probably will, thanks for the heads up. I'll be by a computer in a couple hours here and will release a fix for this 👍

Reply:
Thanks @j178. Fix has been released <3